psql initiation errors

example:
```
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
Error: Process completed with exit code 2.
```

from https://github.com/neurostuff/neurostore/runs/4556822118?check_suite_focus=true

this is an intermittent error

--- a/.github/workflows/workflow.yml
+++ b/.github/workflows/workflow.yml
@@ -32,6 +32,9 @@ jobs:
       name: Create Test Database
       run: |
         docker-compose exec -T \
+        pgsql pg_isready -U postgres -t 10
+
+        docker-compose exec -T \
         pgsql \
         psql -U postgres -c "create database test_db"
     -

